### PR TITLE
fix: broken docs links

### PR DESCRIPTION
**Pull request by vladfrangu**
Stil have a few more to fix, but opening the draft PR now

### CODE REVIEW

File: src/components/InstanceSwitcher/index.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+/**
+ * Only purpose is to cause the website to refresh when switching between "instances",
+ * as a normal redirect from, say, `/platform/actors/tasks` to `/api/client/js/..` will not work by default (since they are
+ * different instances of Docusaurus)
```
Comment (B4nan) [inline src/components/InstanceSwitcher/index.jsx:6]: can we use something from this instead?

https://docusaurus.io/docs/advanced/routing#escaping-from-spa-redirects

Comment (vladfrangu) [reply src/components/InstanceSwitcher/index.jsx:6]: Ohhh, I completely missed that, let me try, if that works thats a MUCH cleaner solution

Comment (vladfrangu) [reply src/components/InstanceSwitcher/index.jsx:6]: It does work, but it opens the link in a new tab, which is not ideal

Comment (B4nan) [reply src/components/InstanceSwitcher/index.jsx:6]: Dang, we don't want that, but we could probably swizzle the Link component and adjust how that works, I already did that for the nav links.

https://github.com/apify/apify-docs-v2/blob/master/apify-docs-theme/src/theme/NavbarItem/NavbarNavLink.jsx#L32-L37

Comment (vladfrangu) [reply src/components/InstanceSwitcher/index.jsx]: I forgot that's a possibility already 😮‍💨. Pushed!

**Review by B4nan — APPROVED** [review]
much better :]

**Pull request merged**